Gemfile

```diff
@@ -0,0 +1,45 @@
+source 'https://rubygems.org'
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '4.0.2'
```

Comment:
Ок) обновлюсь!

2016-05-02 0:44 GMT+06:00 Anton Cherepanov <notifications@github.com>:

> In Gemfile
> <https://github.com/Merff/flashcards/pull/1#discussion_r61690979>:
>
> > @@ -0,0 +1,45 @@
> > +source 'https://rubygems.org'
> > +
> > +# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
> > +gem 'rails', '4.0.2'
>
> @Merff <https://github.com/Merff> почему такая старая версия? 4.2.x
> последняя, следует обновиться =)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Merff/flashcards/pull/1/files/63c399861b1b178b8e864cf0f2a325a375e76c73#r61690979>
>